src/WebJobs.Script/Binding/HttpBinding.cs

```diff
@@ -236,7 +250,10 @@ private static void AddResponseHeader(HttpResponseMessage response, KeyValuePair
                         }
                         break;
                     case "content-md5":
-                        response.Content.Headers.ContentMD5 = header.Value.Value<byte[]>();
+                        if (header.Value is byte[])
```

Comment:
This could also be passed/returned as a string, correct? If so, we should use that as well.